pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -54,6 +63,59 @@ func formatReason(reasons []algorithm.PredicateFailureReason) string {
 	return strings.Join(reasonStrings, ", ")
 }
 
+type predicateEnable struct {
+	memoryPressureEnable bool
+	diskPressureEnable   bool
+	pidPressureEnable    bool
+}
+
+func initPredicateEnable() predicateEnable {
```

Comment:
If all values are `false`, perhaps we do not need this function?

Reply:
I thought it is better to initialize those values.  WIll remove function block.